functions/triangulate/colsog_fn_triangulate_signal.sqf

```diff
@@ -1,12 +1,19 @@
 // 5 minutes timeout between each call.
 private _nextTriangulationTimeSeconds = COLSOG_lastTriangulationTimeSeconds + colsog_triangulation_coolDown;
 private _currentTimeSeconds = serverTime;
+private _nearest_acre_spike = player distance nearestObject [player, "vhf30108spike"]; //Gets the distances to the nearest radio spike
```

Comment:
Would also make the object "vhf30108spike" configurable in the CBA setting in order to be able to use any desired object. This implies renaming variable in order to reflect that.